views/layout.hbs

```diff
@@ -3,10 +3,17 @@
   <head>
   
     <title>{{title}}</title>
-    <link rel='stylesheet' href='/stylesheets/style.css' />
-    <link rel='stylesheet' href='/stylesheets/bootstrap-select.min.css' />
-    <link rel='stylesheet' href='/stylesheets/select2.css' />
-    <link rel='stylesheet' href='/stylesheets/leaguepage.css' />
+    {{#if isProduction}}
+      <link rel='stylesheet' href='/stylesheets/style.css' />
```

Comment:
Isn't this backwards? Don't we want to use the minified resources on production and the main ones on dev?